env0/resource_environment_test.go

```diff
@@ -431,6 +430,152 @@ func TestUnitEnvironmentResource(t *testing.T) {
 			})
 		})
 
+		t.Run("Create and redeploy configuration variables - sensitive values", func(t *testing.T) {
```

Comment:
There were not tests for sensitive configurations.
Added one. Tests both creation and redeploy/update.

Reply:
the test checks that the environment can be created and redeployed with sensitive values? it doesn't check anything regarding the issue at hand, right?

Reply:
The issue was about sensitive values.
Without the fixes in this PR, the test would fail during `plan`.